netlify.toml

```
@@ -0,0 +1,24 @@
+# Netlify build instructions
+[build]
+    command = "./docs/book/install-and-build.sh"
+    publish = "docs/book/book"
+
+# Standard Netlify redirects
+[[redirects]]
+    from = "https://master--kubernetes-sigs-secrets-store-csi-driver.netlify.com/*"
```

```suggestion
    from = "https://master--kubernetes-sigs-secrets-store-csi-driver.netlify.app/*"
```
Netlify no longer uses .com for their addresses - its been switched to .app

---

Updated!